### DISCUSSION TITLE
Scraping after a timestamp

### DISCUSSION THREAD

**Discussion by iankoratskydcn** [kevinzg/facebook-scraper#1071] [Q&A]
Is there a way to scrape only those posts after a specific timestamp? if not, i may try implementing that

**Comment by evildrome** [comment]
No and yes.

Its possible but you need to have scraped all the previous posts.  There is no "jump to" option because the FB infinite scrolling format means you have to pass though all previous posts to get to a specific point.

You can speed things up by just getting posts and ignoring media and comments but you can't start downloading at a specific point.

AFAIK.

**Reply by iankoratskydcn** [reply]
dang
